src/Commands/AbstractDrupalCommands.php

```diff
@@ -316,8 +317,46 @@ public function drushSetup(array $options = [
     }
 
     /**
-     * Setup Drupal settings overrides.
+     * Setup Drupal services file.
+     *
+     * This command will add the configuration under service_parameters present in
+     * runner.yml into "services.yml" file in the site directory.
+     *
+     * @command drupal:services-setup
+     *
+     * @option root         Drupal root.
+     * @option sites-subdir Drupal site subdirectory.
+     * @option force        Drupal force generation of a new services.yml.
+     *
+     * @param array $options
      *
+     * @return \Robo\Collection\CollectionBuilder
+     */
+    public function servicesSetup(array $options = [
+        'root' => InputOption::VALUE_REQUIRED,
+        'sites-subdir' => InputOption::VALUE_REQUIRED,
+        'force' => false,
+    ])
+    {
+        // Read given parameters.
+        $service_parameters['parameters'] = $this->getConfig()->get('drupal.service_parameters', []);
+        $dumper = new Dumper(2);
+        $yaml = $dumper->dump($service_parameters, PHP_INT_MAX, 0, Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE);
+
+        // Set the destination file.
+        $services_destination_file = $options['root'] . '/sites/' . $options['sites-subdir'] . '/services.yml';
+
+        $collection = [];
+        if (true === (bool) $options['force'] || !file_exists($services_destination_file)) {
```

Comment:
` $options['force']` is either `true` or `false`, Robo internals make sure of that, we don't need casting.

Reply:
Done.